tensorboard/uploader/proto/tensor.proto

```diff
@@ -0,0 +1,29 @@
+syntax = "proto3";
+
+package tensorboard.service;
+
+import "google/protobuf/timestamp.proto";
+import "tensorboard/compat/proto/summary.proto";
+import "tensorboard/compat/proto/tensor.proto";
+
+// One point viewable on a tensor metric plot.
+message TensorPoint {
+  // Step index within the run.
+  int64 step = 1;
+  // Timestamp of the creation of this point.
+  google.protobuf.Timestamp wall_time = 2;
+  // Value of the point at this step / timestamp.
+  .tensorboard.TensorProto value = 3;
+}
+
+// Metadata for the TensorPoints stored for one (Experiment, Run, Tag).
+message TensorPointMetadata {
```

Comment:
Is this used anywhere?  It appears forked from ScalarPointMetadata, which is also unused?

Reply:
Its used in the storage backend, but defined here.